modules/platform/nuxeo-platform-document-routing/nuxeo-routing-rest-api/src/test/java/org/nuxeo/ecm/restapi/server/RoutingRestBaseTest.java

```diff
@@ -119,6 +126,15 @@ protected String getCreateAndStartWorkflowBodyContent(String workflowName, List<
             result.append("]");
         }
 
+        if (variables != null && !variables.isEmpty()) {
+            result.append(", \"variables\": {");
+            result.append(variables.entrySet()
+                                   .stream()
+                                   .map(e -> "\"" + e.getKey() + "\": \"" + e.getValue() + "\"")
```

Comment:
This seems brittle, what if there's a `"` in the value?

Reply:
It fails :)

Reply:
Ah j'avais pas vu que c'était dans les tests...